src/components/uikit/checkbox/index.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = React.InputHTMLAttributes<HTMLInputElement> & {
+  label: React.ReactNode;
+};
+
+export const Checkbox = (props: Props) => {
+  const { label, ...restProps } = props;
+  return (
+    <label className="... flex gap-4">
```

Comment:
Looks like you used my example from tasks. Please, make sure that you use only necessary code.
```suggestion
    <label className="flex gap-4">
```